src/olympia/devhub/tests/test_tasks.py

```diff
@@ -886,3 +886,27 @@ def test_returns_received_results(self):
         results = {'errors': 1}
         returned_results = tasks.forward_linter_results(results, 123)
         assert results == returned_results
+
+
+class TestCreateSitePermissionVersion(TestCase):
+    @mock.patch('olympia.devhub.tasks.SitePermissionVersionCreator')
+    def test_pass_down_to_creator_util(self, SitePermissionVersionCreator_mock):
+        user = user_factory()
+        tasks.create_site_permission_version(
+            user_pk=user.pk,
+            remote_addr='127.0.0.42',
+            install_origins=['https://example.com'],
+            site_permissions=['foo'],
+        )
+        assert SitePermissionVersionCreator_mock.call_count == 1
+        assert SitePermissionVersionCreator_mock.call_args[0] == ()
```

Comment:
What is this? Is the the `self` that is always passed into functions?

Reply:
Regular (non-keyword) arguments. There are none, everything is passed as keyword arguments.